.gitignore

```diff
@@ -63,3 +63,7 @@ __init__.pyi
 
 # Logs
 *.log
+
+# Lightbulb and test bot
+lightbulb/
+test-bot.py
```

Comment:
Could you please remove these? They have nothing to do with the project

Reply:
i put them here just for testing, i will remove it if you want to

Reply:
Yes please